packages/frontend/src/components/LightdashVisualization/VisualizationProvider.tsx

```diff
@@ -117,6 +119,10 @@ export const VisualizationProvider: FC<Props> = ({
     const { validBigNumberConfig } = bigNumberConfig;
     const { validTableConfig } = tableConfig;
 
+    const isSqlRunner = useMemo(() => {
+        return explore?.name === 'sql_runner';
+    }, [explore?.name]);
```

Comment:
some time ago I realised that memo is useless unless there is a computation which actually takes time. this should be slower than without memo.

Reply:
mmm, I have no idea, I know that sometimes the compiler force us to use memo on hooks because otherwise it thinks this is recalculated all the time... I'm happy to revisit all the memos if you think they are useless and inefficient.

Reply:
no need to rewrite. I have a benchmark somewhere and will share in eng